docs/user_manual.md

```diff
@@ -167,204 +479,181 @@ List of supported formats:
+- KITTI Point Clouds
 
 To create an unlabelled dataset from an arbitrary directory with images use
-`ImageDir` format:
+`image_dir` and `image_zip` formats:
 
-```bash
+``` bash
 datum create -o <project/dir>
-datum add path -p <project/dir> -f image_dir <directory/path/>
+datum add -p <project/dir> -f image_dir <directory/path/>
 ```
 
-or if you work with Datumaro API:
+or, if you work with Datumaro API:
 
-For using with a project:
+- for using with a project:
 
-```python
-from datumaro.components.project import Project
+  ```python
+  from datumaro.components.project import Project
 
-project = Project()
-project.add_source('source1', {
-  'format': 'image_dir',
-  'url': 'directory/path/'
-})
-dataset = project.make_dataset()
-```
+  project = Project.init()
+  project.import_source('source1', format='image_dir', url='directory/path/')
+  dataset = project.working_tree.make_dataset()
+  ```
 
-And for using as a dataset:
+- for using as a dataset:
 
-```python
-from datumaro.components.dataset import Dataset
+  ```python
+  from datumaro.components.dataset import Dataset
 
-dataset = Dataset.import_from('directory/path/', 'image_dir')
-```
+  dataset = Dataset.import_from('directory/path/', 'image_dir')
+  ```
 
 This will search for images in the directory recursively and add
 them as dataset entries with names like `<subdir1>/<subsubdir1>/<image_name1>`.
-The list of formats matches the list of supported image formats in OpenCV.
+The list of formats matches the list of supported image formats in OpenCV:
 ```
 .jpg, .jpeg, .jpe, .jp2, .png, .bmp, .dib, .tif, .tiff, .tga, .webp, .pfm,
 .sr, .ras, .exr, .hdr, .pic, .pbm, .pgm, .ppm, .pxm, .pnm
 ```
 
-After addition into a project, images can be split into subsets and renamed
-with transformations, filtered, joined with existing annotations etc.
+Once there is a `Dataset` instance, it's items can be split into subsets,
+renamed, filtered, joined with annotations, exported in various formats etc.
 
-To use a video as an input, one should either [create an Extractor plugin](../docs/developer_guide.md#plugins),
+To use a video as an input, one should either create a [plugin](#extending),
 which splits a video into frames, or split the video manually and import images.
 
-## Command line workflow
+## Command reference
 
-The key object is a project, so most CLI commands operate on projects.
-However, there are few commands operating on datasets directly.
-A project is a combination of a project's own dataset, a number of
-external data sources and an environment.
-An empty Project can be created by `project create` command,
-an existing dataset can be imported with `project import` command.
-A typical way to obtain projects is to export tasks in CVAT UI.
+![cli](images/datum_cli.svg)
 
-If you want to interact with models, you need to add them to project first.
+> **Note**: command invocation syntax is subject of change,
+> **always refer to the `--help` output of the specific command**
```

Comment:
It's unclear to me what this recommendation implies. Are you trying to say that the documentation may not match the actual syntax of the command?

Reply:
I meant that documentation can be outdated. The code is updated more often, CLI help must always be actual.